receiver.py

```diff
@@ -180,15 +181,15 @@ def data_lines(self):
                     print("command {}".format(' '.join("0x{:02x}".format(i) for i in cmd)))
                     print("sent a command")
                 self.outbound = []
-            except (OSError, serial.SerialException):
```

Comment:
The error message logged here is specific. If you want to capture more errors, the you should aa second catch block.